stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -254,11 +254,19 @@ end
     end
 end
 
+Base._reverse(A::Symmetric, dims::Integer) = (reverse((Matrix(A)); dims))
```

Comment:
```suggestion
Base._reverse(A::Symmetric, dims::Integer) = reverse!(Matrix(A); dims)
```
the extra parens were not needed, and you can use `reverse!` as noted above.